edx_lint/tamper_evident.py

```diff
@@ -34,6 +34,10 @@ def write(self, text, hashline=b"# {}"):
                 to the file, with "{}" replaced with the hash.
 
         """
+        # Ensure we are working with bytes.
+        if isinstance(text, str):
+            text = str.encode(text)
```

Comment:
The argument being passed when running the command is of type `str`: https://github.com/edx/edx-lint/blob/master/edx_lint/cmd/write.py#L123. Perhaps a better solution is to update this method to accept `str` instead of `bytes`?